src/TLuaInterpreter.cpp

```diff
@@ -3265,202 +3265,322 @@ int TLuaInterpreter::setBackgroundImage( lua_State *L )
 
 int TLuaInterpreter::setLabelClickCallback( lua_State *L )
 {
-    string luaSendText="";
-    if( ! lua_isstring( L, 1 ) )
-    {
-        lua_pushstring( L, "setLabelClickCallback: wrong argument type" );
+    Host * pHost = TLuaInterpreter::luaInterpreterMap[L];
+    if(! pHost) {
+        lua_pushstring( L, tr( "setLabelClickCallback: NULL Host pointer - something is wrong!" )
+                        .toUtf8().constData() );
         lua_error( L );
         return 1;
     }
-    else
-    {
-        luaSendText = lua_tostring( L, 1 );
-    }
-    string luaName="";
-    if( ! lua_isstring( L, 2 ) )
-    {
-        lua_pushstring( L, "setLabelClickCallback: wrong argument type" );
+
+    QString labelName;
+    if( ! lua_isstring( L, 1 ) ) {
+        lua_pushstring( L, tr( "setLabelClickCallback: bad argument #1 type (label name as string expected, got %1!)" )
```

Comment:
I'm starting to be concerned that we're getting too much into a guessing game of what we think the users want and don't want. Let's put out a Mudlet that can actually work with other languages first and translate the GUI because those are obvious desired features, and leave the rest - such as enablement of API translation - for later, just like it is on the roadmap. Once we get Mudlet into the hands of non-English users, we'll see given mass feedback if it's necessary.

tl:dr; lets not clutter (and slow down with unneeded hash lookups) mudlet with ``tr`` everywhere right now except GUI and we'll revisit this in the future.